.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,10 +31,15 @@ If you believe your functionality is valuable to the entire Qwik Community, disc
 * Is it related to any opened issues? (please add them here)
 -->
 
-# Checklist:
+# Checklist
+
+<!--
+* delete the items that are not relevant, so it's easy to tell if the PR is ready to be merged
+* add items that are relevant and need to be done before merging
+-->
 
 - [ ] My code follows the [developer guidelines of this project](https://github.com/QwikDev/qwik/blob/main/CONTRIBUTING.md)
-- [ ] I have performed a self-review of my own code
-- [ ] I have ran `pnpm change` and documented my changes
-- [ ] I have made corresponding changes to the Qwik docs
+- [ ] Performed a self-review of my own code
+- [ ] Added changesets with `pnpm change`
+- [ ] Made corresponding changes to the Qwik docs
```

Comment:
previous wording was clearer to me. It's mostly for people that are not contributing often so better be clear

Reply:
yes 👍